Remotion/Web/Core/Res/Themes/NovaViso/Image/sprite.svg

```diff
@@ -9,6 +9,11 @@
         <polyline points="6.5 11 9.5 8 6.5 5" />
       </g>
     </g>
+    <g id="SingleChevronWhite">
+      <g stroke="#ffffff" stroke-width="1.2" fill="none" fill-rule="evenodd" stroke-linecap="square" stroke-linejoin="round">
```

Comment:
Quick question to @iam-mholle: can we override the color inside the svg file, i.e. have a generic black chevron and then make it white when we use it in line 44?

Reply:
It should be possible to style it via the css section in the svg file. Not 100% sure tho.

Reply:
@iam-mholle thanks!
@a-ctor I think we should try this timeboxed to remove duplication. Should it make things much more complicated, we can revisit, of course. But the color-shift may also be relevant for making disabled icons, etc, so we may need this anyway in the (very near) future.

Reply:
It is probably easiest to apply a filter to the image element and have just one black item. For example, in the this case we just need a `filter: invert()` to get to the white chevron. Other colors can be created using more complex filters.

Reply:
okay, then we stick with the status quo for now. Using a filter for this feels like overkill.